src/cosmos-ledger.ts

```diff
@@ -212,3 +215,16 @@ export const checkAppMode = (testModeAllowed: Boolean, testMode: Boolean) => {
     )
   }
 }
+
+// doesn't properly work in ledger-cosmos-js
+function getBech32FromPK(hrp, pk) {
+  if (pk.length !== 33) {
+    throw new Error('expected compressed public key [31 bytes]')
```

Comment:
what does this error message mean?

Reply:
I copied that from Juan
```suggestion
    throw new Error('The private key needs to be of exactly 33 bytes length to be used for a Cosmos address.')
```

Reply:
ok that's better - but is the error actionable for the user? if a user sees this error, what are they to do?

Reply:
This should never be user facing

Reply:
perfect!